pkgs/servers/dendrite/default.nix

```diff
@@ -3,17 +3,20 @@
 
 buildGoModule rec {
   pname = "matrix-dendrite";
-  version = "0.9.3";
+  version = "0.9.4";
 
   src = fetchFromGitHub {
     owner = "matrix-org";
     repo = "dendrite";
     rev = "v${version}";
-    sha256 = "sha256-A/4FN0FYuhP5AO7yQq/o5ZJKEJAotfervot70Scgj6M=";
+    sha256 = "sha256-UB51Rd0AWEfj6qTfrRSWK/dq9MdLCqYoR/Gjwf65ZQk=";
   };
 
   vendorSha256 = "sha256-tgVImIfn1lPTYGXczoAxVta3L+VR0v13KowLIYQ7bwY=";
 
+  # some tests are racy, re-enable once upstream has fixed them
```

Comment:
We're running into this on [Hydra](https://hydra.nixos.org/build/187746682), and this release includes [a fix for it](https://github.com/matrix-org/dendrite/pull/2654). However, after talking to upstream, there seem to be at least one test that is still racy. I considered removing just that one test, like we do already, but what if another racy test crops up, are we just going to keep playing whack-a-mole?

Open to thoughts on this, I could totally be thinking about this the wrong way.

Reply:
Skipping tests in go is a bit difficult, so let's just go with this approach.

Reply:
Do you remember which test was still racy, @winterqt?